github4s/src/main/scala/github4s/domain/PullRequest.scala

```diff
@@ -141,3 +141,13 @@ final case object PRREventComment        extends PullRequestReviewEvent("COMMENT
 final case object PRREventPending        extends PullRequestReviewEvent("PENDING")
 
 case class CreateReviewComment(path: String, position: Int, body: String)
+
+final case class RequestedReviewersRequest(
```

Comment:
nit: since it's used for both adding and removing reviewers maybe `ReviewersRequest` is a better name?